src/params.rs

```diff
@@ -72,7 +73,7 @@ impl<'a> Iterator for Params<'a> {
                 self.bound_types.clear();
                 for i in 0..self.params as usize {
                     self.bound_types.push((
-                        myc::constants::ColumnType::from(typmap[2 * i as usize]),
+                        myc::constants::ColumnType::try_from(typmap[2 * i as usize]).unwrap(),
```

Comment:
We should arguably change `Item` here to `Result<Item>` to propagate errors from here, but can do that as part of a breaking change. In the meantime, would you mind changing this to
```rust
.ok_or_else(|| panic!("got unsupported column type 0x{:x}", typmap[2 * i as usize]))
```
so that the error a user might get is a little more helpful?

Reply:
Done.